### ISSUE TITLE
Confusing meeting times on website

### ISSUE THREAD

**Issue by jwulf** [gluster/glusterdocs#527]
There are two conflicting times for the community meetings on the website. One says Tuesday, the other says Wednesday:

<img width="470" alt="Screen Shot 2020-01-25 at 12 21 04 pm" src="https://user-images.githubusercontent.com/406975/73120363-492ea400-3f6d-11ea-8c2a-81ab180943a0.png">

<img width="338" alt="Screen Shot 2020-01-25 at 12 20 58 pm" src="https://user-images.githubusercontent.com/406975/73120359-3f0ca580-3f6d-11ea-9c0b-d6617b832538.png">

**Comment by aravindavk** [comment]
@harigowtham ^^

**Comment by sankarshanmukhopadhyay** [comment]
The link at https://www.gluster.org/community/ has the set of meetings which are now in place. There are twice-a-month meetings on Tuesdays aimed at 3 timezones. In the near term it is likely that we'll find a single meeting time slice to enable overlap between these TZs (there is a proposal on the list).

Could you help me with the URL which has the image for "development" and has the Wednesday in the text?

**Comment by aravindavk** [comment]
https://www.gluster.org/community/ in "Development" section

**Comment by harigowtham** [comment]
Hi Aravinda, I'm looking into it. Will get it fixed soon

**Comment by sankarshanmukhopadhyay** [comment]
If the meeting times are indeed to be changed as Yati P had mentioned on lists, we could wait and do a single time update to the current decision.

**Comment by harigowtham** [comment]
Yes, I'll wait for the confirmation to the time change we are looking into
